### PR TITLE
chore: Bump versions 

### PR DESCRIPTION
**Pull request by GordonSmith**
## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: components/rollup.config.js [hunk 1]
```diff
@@ -28,71 +28,76 @@ const parserOptions = {
     sourceType: "module",
 };
 
-export const plugins = [
+console.log(Object.keys(terser));
```
Comment (GordonSmith) [inline components/rollup.config.js:31]: Remove console.log

File: packages/dataflow/package.json [hunk 2]
```diff
@@ -22,12 +22,12 @@
...
     ],
     "scripts": {
-        "clean": "rimraf lib* types dist *.tsbuildinfo",
+        "clean": "rimraf --glob lib* types dist *.tsbuildinfo",
         "compile-es6": "tsc --module es6 --outDir ./lib-es6",
         "compile-es6-watch": "npm run compile-es6 -- -w",
         "compile-umd": "tsc --module umd --outDir ./lib-umd",
         "compile-umd-watch": "npm run compile-umd -- -w",
-        "bundle": "rollup -c",
+        "bundle": "rollup -c --bundleConfigAsMjs",
```
Comment (GordonSmith) [inline packages/dataflow/package.json:30]: Is bundleConfigAsMjs needed?

**Review by jeclrsg — APPROVED** [review]
@GordonSmith a few lines of commented out code in a test was the only other thing I noticed

File: packages/dgrid/src/__tests__/test1.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                ["W", 14, 21, 18, 18],
+                ["X", 15, 23, 21, 18],
+                ["Y", 21, 23, 23, 21],
+                ["Z", 23, 24, 24, 24]
+            ])
+            .on("click", (row, column, selected) => {
+                console.info("click", row, column, selected);
+            })
+            ;
+        // setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/dgrid/src/__tests__/test1.ts:41]: Actually I want to keep that...

**Pull request merged**